samcli/local/common/runtime_template.py

```diff
@@ -99,17 +102,27 @@
     itertools.chain(*[c["runtimes"] for c in list(itertools.chain(*(RUNTIME_DEP_TEMPLATE_MAPPING.values())))])
 )
 
+# Order here should be a the group of the latest versions of runtimes followed by runtime groups
 INIT_RUNTIMES = [
-    "nodejs10.x",
-    "python3.7",
+    # latest of each runtime version
```

Comment:
I think this list is what is used to showcases the runtimes during `init`, we should make this ordered, with a set maybe.

Reply:
This is ordered just in a list. I can update it to a set but there isn't a whole difference here (other than duplicate entry guard).

This is used within interactive init not in the help text.